LQResultProducer.py

```diff
@@ -109,19 +111,36 @@
 #eleHEEPScale = '*((1-IsMuon_muon1)*(((Eta_muon1>-2.5)*(Eta_muon1<-1.566)*0.983)+((Eta_muon1>-1.4442)*(Eta_muon1<-0.5)*0.971)+((Eta_muon1>-0.5)*(Eta_muon1<-0.0)*0.961)+((Eta_muon1>0.0)*(Eta_muon1<0.5)*0.973)+((Eta_muon1>0.5)*(Eta_muon1<1.4442)*0.978)+((Eta_muon1>1.566)*(Eta_muon1<2.5)*0.980))+(1-IsMuon_muon2)*(((Eta_muon2>-2.5)*(Eta_muon2<-1.566)*0.984)+((Eta_muon2>-1.4442)*(Eta_muon2<-0.5)*0.971)+((Eta_muon2>-0.5)*(Eta_muon2<-0.0)*0.961)+((Eta_muon2>0.0)*(Eta_muon2<0.5)*0.973)+((Eta_muon2>0.5)*(Eta_muon2<1.4442)*0.978)+((Eta_muon2>1.566)*(Eta_muon2<2.5)*0.980)))'
 eleHEEPScale = '*((1-IsMuon_muon1)*(((abs(Eta_muon1)>0.0)*(abs(Eta_muon1)<0.5)*0.967)+((abs(Eta_muon1)>0.5)*(abs(Eta_muon1)<1.4442)*0.975)+((abs(Eta_muon1)>1.566)*(abs(Eta_muon1)<2.5)*0.983))+(1-IsMuon_muon2)*((abs(Eta_muon2)>0.0)*(abs(Eta_muon2)<0.5)*0.967+((abs(Eta_muon2)>0.5)*(abs(Eta_muon2)<1.4442)*0.975)+((abs(Eta_muon2)>1.566)*(abs(Eta_muon2)<2.5)*0.983)))'
 
-#BTAG scale factors
-#2016
-bTag1SFmedium = '*(1-(1-(DeepJet_jet1>0.3033)*bTagSF_jet1)*(1-(DeepJet_jet2>0.3033)*bTagSF_jet2))' #fixme btag -> BTag
-bTag1SFmediumUp = '*(1-(1-(DeepJet_jet1>0.3033)*bTagSF_jet1Up)*(1-(DeepJet_jet2>0.3033)*bTagSF_jet2Up))'
-bTag1SFmediumDown = '*(1-(1-(DeepJet_jet1>0.3033)*bTagSF_jet1Down)*(1-(DeepJet_jet2>0.3033)*bTagSF_jet2Down))'
-
-bTagsel1medium = '*(((DeepJet_jet1>0.3033)+(DeepJet_jet2>0.3033))>0)'
+#BTAG scale factors and selection
+deepJetWPmedium = '1'
+if year == '2016': deepJetWPmedium = '0.3093'
+elif year == '2017': deepJetWPmedium = '0.3033'
+elif year == '2018': deepJetWPmedium = '0.2770'
+
+if btags == '0':
+	bTagSFmedium = '*1'
+	bTagSFmediumUp = '*1'
+	bTagSFmediumDown = '*1'
+	bTagselmedium = '*1'
```

Comment:
The 0 btag case will need a b veto, i.e.

bTagselmedium = '*(((DeepJet_jet1>'+deepJetWPmedium+')+(DeepJet_jet2>'+deepJetWPmedium+'))==0)'

We will need to think about whether this requires an event weight, have to check the twikis

Reply:
I see. So it would be exactly 0 b-tags, at least 1 b-tag, or exactly 2 b-tags (since we only consider the 2 leading jets).

Reply:
For now, yes. Eventually it should be ==0, ==1, ==2